src/Core/Settings.cpp

```diff
@@ -2391,7 +2391,7 @@ What to do when the limit is exceeded.
     DECLARE(UInt64, max_bytes_before_external_group_by, 0, R"(
 If memory usage during GROUP BY operation is exceeding this threshold in bytes, activate the 'external aggregation' mode (spill data to disk). Recommended value is half of the available system memory.
 )", 0) \
-    DECLARE(Double, max_bytes_ratio_before_external_group_by, 0., R"(
+    DECLARE(Double, max_bytes_ratio_before_external_group_by, 0.5, R"(
```

Comment:
@Michicosun https://github.com/ClickHouse/ClickHouse/pull/71406/files#diff-bd80c4328eff8a0779b6f7227e8d6d699b1497088e5883becf218e21d955b3b1 stated that the internals are subject to change. Since the ratio setting is now the default, let's remove the note I referenced?

Reply:
Sure, totally forgot about those notes.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/74000